.github/workflows/releases.yaml

```diff
@@ -0,0 +1,46 @@
+name: Releases
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  PUSH: 'true'
+  DIRECT: 'true'
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
seems it's not used?

Reply:
`gh release upload` will use `GH_TOKEN` to put `binary` into the release page.